app/messaging/get-pending-payment-requests.js

```diff
@@ -15,6 +15,11 @@ const getPendingPaymentRequests = async (transaction) => {
     },
     order: ['paymentRequestId']
   })
+  return paymentRequests.map(x => x.get({ plain: true })).map(removeNullProperties)
```

Comment:
Mapping object to a `plain` Sequelize object will all Sequelize properties stripped out.  This can be done by setting both `raw` and `nest` properties to `true`.  However, due to an issue in Sequelize, this would result in the `invoiceLines` property being converted to an object with only the first line rather than all of them in an array which would result in incorrect payments.

Also adding function to remove all properties set to `null` in the database as they are not required in downstream services.